tests/test_logger.py

```diff
@@ -0,0 +1,25 @@
+import json
+import logging
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from supertokens_python.constants import VERSION
+from supertokens_python.logger import streamFormatter, streamHandler
+
+
+class LoggerUtilsTests(TestCase):
+    @patch('supertokens_python.logger._get_log_timestamp', return_value='timestamp')
+    def test_json_msg_format(self, mock_timestamp: MagicMock):  # pylint: disable=unused-argument
```

Comment:
You can use `_` as an argument name instead of using `# pylint: disable=unused-argument`.

Reply:
Thanks! Fixed it.

I tried it last time but I think my python extension didn't respond fast enough and that made me believe that this doesn't work.